packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -452,14 +452,16 @@ contract DeployImplementations is Script {
     // --- Fault Proofs Contracts ---
 
     // The fault proofs contracts are configured as follows:
-    //   - DisputeGameFactory: Proxied, bespoke per chain.
-    //   - AnchorStateRegistry: Proxied, bespoke per chain.
-    //   - FaultDisputeGame: Not proxied, bespoke per chain.
-    //   - PermissionedDisputeGame: Not proxied, bespoke per chain.
-    //   - DelayedWETH: Proxied, and two bespoke ones per chain (one for each DisputeGame).
-    //   - PreimageOracle: Not proxied, shared by all standard chains.
-    //   - MIPS: Not proxied, shared by all standard chains.
-    //   - OptimismPortal2: Proxied, shared by all standard chains.
+    // | Contract                | Proxied | Deployment                        | MCP Ready  |
+    // |-------------------------|---------|-----------------------------------|------------|
+    // | DisputeGameFactory      | Yes     | Bespoke                           | Yes        |
+    // | AnchorStateRegistry     | Yes     | Bespoke                           | No         |
+    // | FaultDisputeGame        | No      | Bespoke                           | No         |
+    // | PermissionedDisputeGame | No      | Bespoke                           | No         |
+    // | DelayedWETH             | Yes     | Two bespoke (one per DisputeGame) | No         |
+    // | PreimageOracle          | No      | Shared                            | N/A        |
+    // | MIPS                    | No      | Shared                            | N/A        |
+    // | OptimismPortal2         | Yes     | Shared                            | N/A        |
```

Comment:
Similar to above comment, OptimismPortal2 is MCP iff all standard chains are required to share the same values on immutables

Reply:
OK. This at least shouldn't say `N/A`. That's meant for singletons.